extensions/wikia/TemplateDraft/controllers/TemplateDraftController.class.php

```diff
@@ -35,4 +35,23 @@ public function createDraftContent( Title $title, $content, Array $flags ) {
 
 		return $newContent;
 	}
+
+	public function markTemplateAsNotInfobox() {
```

Comment:
This is a state changing action, so should require a post and a token as a best practice.